app/controllers/api/vms_controller.rb

```diff
@@ -10,7 +10,7 @@ class VmsController < BaseController
     include Subcollections::MetricRollups
 
     VALID_EDIT_ATTRS = %w(description child_resources parent_resource).freeze
-    RELATIONSHIP_COLLECTIONS = [:vms, :templates].freeze
+    RELATIONSHIP_COLLECTIONS = %w(vms templates).freeze
```

Comment:
Does the change from symbol to string matter?

Reply:
It's only referenced [here](https://github.com/ManageIQ/manageiq-api/pull/97/files/5cfadbf45766890b6a86b78f35188bd23186cea5#diff-032d88eafbded3d355900924d1239226R293)...I changed it because every other consumer of `Href.resource` doesn't care because it's either intepolated or converted by the caller to a symbol anyway, so it helped simplify.